### PR TITLE
docs: Document how to generate and push to kubewarden/k8s-objects

### PR DESCRIPTION
**Pull request by viccuad**
## Description

Relates to https://github.com/kubewarden/kubewarden-controller/issues/794.

## Test

Used this steps to create branch:
https://github.com/kubewarden/k8s-objects/tree/release-1.30
and tag:
https://github.com/kubewarden/k8s-objects/tree/v1.30.0-kw1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -142,3 +142,37 @@ directory. This is the only directory you need to preserve.
 
 > **Note:** the name of the final Git repository can be changed using the `-repo`
 > flag.
+
+### Pushing new versions to kubewarden/k8s-objects
+
+Set `KUBERNETES_VERSION_MIN`, `KUBERNETES_VERSION_MAX` in `mass-generate.sh`,
+`KUBEMINOR` in `mass-push.sh` as needed.
+
+Delete and reclone kubewarden/k8s-objects. In my case:
```
Comment (flavio) [inline README.md:151]: You don't have to do that, you just need the repo checked out locally.

File: README.md [hunk 2]
```diff
@@ -142,3 +142,37 @@ directory. This is the only directory you need to preserve.
...
+git push origin release-1.30
+git push origin v1.30.0-kw1
+```
+
+Or do a mass push, potentially overwritting old branches:
+
+```console
+mv ~/suse/kw/k8s-objects ~/hacking/kubernetes/kubewarden/k8s-objects
+./mass-push.sh -m commit.md --git-dir ~/suse/kw/k8s-objects
+```
```
Comment (flavio) [inline README.md:178]: The mass push is safe to be done, there's no overwriting involved. Also, it doesn't take a `-m` parameter

**Review by flavio — CHANGES_REQUESTED** [review]
Thanks for having documented all of that. I left some comments to be addressed

File: README.md (hunk 1 above)
Comment (viccuad) [reply README.md:151]: Amended.

File: README.md (hunk 2 above)
Comment (viccuad) [reply README.md:178]: Amended.

**Review by fabriziosestito — APPROVED** [review]

**Pull request merged**